api/logs/views.py

```diff
@@ -220,7 +221,7 @@ class NodeLogContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin,
     required_read_scopes = [CoreScopes.USERS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    serializer_class = UserSerializer
+    serializer_class = UnregisteredContributorSerializer
```

Comment:
Thanks again for the explanation-- clearly there's some history here that I missed.

I really don't want to block this PR, and would be OK with a comment explaining the need for `UnclaimedUserSerializer`.

But IMO doing away with such a specific endpoint seems like a great idea to me. Probably needs approval from @brianjgeiger to go forward here.